cyclops-ui/src/components/k8s-resources/common/PodTable/PodLogs.tsx

```diff
@@ -71,6 +72,15 @@ const PodLogs = ({ pod }: PodLogsProps) => {
                   logs.length === 0 ? "No logs available" : logs.join("\n")
                 }
                 readOnly={true}
+                commands={[
+                  {
+                    name: "showSearchBox",
+                    bindKey: { win: "Ctrl-F", mac: "Command-F" },
+                    exec: function (editor) {
+                      editor.execCommand("showSearchBox");
+                    },
+                  },
+                ]}
```

Comment:
Can we remove this? I think importing ace-builds searchbox does the job and binds it to `Ctrl-f|Command-f` -> https://github.com/ajaxorg/ace-builds/blob/master/src/ext-searchbox.js

Reply:
That's right!! it does, updated the code checkout the latest commit.